docs/sanic/routing.md

```diff
@@ -41,24 +41,61 @@ inside the quotes. If the parameter does not match the specified type, Sanic
 will throw a `NotFound` exception, resulting in a `404: Page not found` error
 on the URL.
 
+### Supported types
+
+* string
+    * "Bob" or "Python 3" both would work here
+* int
+    * 10, 20, 30, -10 all work here
+    * No floats work here
+* number
+    * 1, 1.5, 10, -10 all work here
+* alpha
+    * "Bob", "Python" all work here
+    * If it contains a symbol or a non alphanumeric character it will fail
+* path
+    * "hello", "hello.text" and "hello world" all work here
+* uuid
+    * 123a123a-a12a-1a1a-a1a1-1a12a1a12345 and all other UUIDv4 work here
+* regex expression
+
+If no type is set then a string is expected. The argument given to the function will
+    always be a string, independent of the type.
+
```

Comment:
Maybe also make a specific call out that it is `string` and not `str`?

Reply:
Tweaked to bring that up